<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTextEdit" name="textEdit">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>1</y>
      <width>801</width>
      <height>531</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu_F">
    <property name="title">
     <string>檔案(F)</string>
    </property>
    <addaction name="action_C"/>
    <addaction name="action_C_2"/>
    <addaction name="separator"/>
    <addaction name="action_S"/>
    <addaction name="action_A"/>
    <addaction name="separator"/>
    <addaction name="action_X"/>
   </widget>
   <widget class="QMenu" name="menu_E">
    <property name="title">
     <string>編輯(E)</string>
    </property>
    <addaction name="action_U"/>
    <addaction name="action_T"/>
    <addaction name="action_C_3"/>
    <addaction name="action_A_2"/>
    <addaction name="action_P"/>
   </widget>
   <addaction name="menu_F"/>
   <addaction name="menu_E"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="action_N"/>
   <addaction name="action_C"/>
   <addaction name="action_S"/>
   <addaction name="action_A"/>
   <addaction name="separator"/>
   <addaction name="action_U"/>
   <addaction name="action_T"/>
   <addaction name="action_C_3"/>
   <addaction name="action_P"/>
  </widget>
  <action name="action_N">
   <property name="icon">
    <iconset resource="images/images/designwindow.qrc">
     <normaloff>:/images/new.png</normaloff>:/images/new.png</iconset>
   </property>
   <property name="text">
    <string>新增(N)</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+S</string>
   </property>
  </action>
  <action name="action_C">
   <property name="icon">
    <iconset resource="images/images/designwindow.qrc">
     <normaloff>:/images/open.png</normaloff>:/images/open.png</iconset>
   </property>
   <property name="text">
    <string>開啟(&amp;O)</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+O</string>
   </property>
  </action>
  <action name="action_C_2">
   <property name="text">
    <string>關閉(&amp;C)</string>
   </property>
  </action>
  <action name="action_S">
   <property name="icon">
    <iconset resource="images/images/designwindow.qrc">
     <normaloff>:/images/save.png</normaloff>:/images/save.png</iconset>
   </property>
   <property name="text">
    <string>儲存(&amp;S)</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+S</string>
   </property>
  </action>
  <action name="action_A">
   <property name="icon">
    <iconset resource="images/images/designwindow.qrc">
     <normaloff>:/images/asave.png</normaloff>:/images/asave.png</iconset>
   </property>
   <property name="text">
    <string>另存(&amp;A)</string>
   </property>
  </action>
  <action name="action_X">
   <property name="text">
    <string>退出(&amp;X)</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+X</string>
   </property>
  </action>
  <action name="action_U">
   <property name="icon">
    <iconset resource="images/images/designwindow.qrc">
     <normaloff>:/images/undo.png</normaloff>:/images/undo.png</iconset>
   </property>
   <property name="text">
    <string>還原(&amp;U)</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+U</string>
   </property>
  </action>
  <action name="action_T">
   <property name="icon">
    <iconset resource="images/images/designwindow.qrc">
     <normaloff>:/images/cut.png</normaloff>:/images/cut.png</iconset>
   </property>
   <property name="text">
    <string>剪下(&amp;T)</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+T</string>
   </property>
  </action>
  <action name="action_C_3">
   <property name="icon">
    <iconset resource="images/images/designwindow.qrc">
     <normaloff>:/images/copy.png</normaloff>:/images/copy.png</iconset>
   </property>
   <property name="text">
    <string>複製(&amp;C)</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+C</string>
   </property>
  </action>
  <action name="action_A_2">
   <property name="text">
    <string>全選(&amp;A)</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+A</string>
   </property>
  </action>
  <action name="action_P">
   <property name="icon">
    <iconset resource="images/images/designwindow.qrc">
     <normaloff>:/images/paste.png</normaloff>:/images/paste.png</iconset>
   </property>
   <property name="text">
    <string>貼上(&amp;P)</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+P</string>
   </property>
  </action>
 </widget>
 <resources>
  <include location="images/images/designwindow.qrc"/>
 </resources>
 <connections/>
</ui>
